quic/s2n-quic/src/provider/event/console_perf.rs

```diff
@@ -0,0 +1,449 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::provider::event;
+use core::time::Duration;
+use std::{
+    sync::{
+        atomic::{AtomicU64, Ordering},
+        Arc,
+    },
+    time::{SystemTime, UNIX_EPOCH},
+};
+
+/// An event subscriber that prints performance metrics to the console via STDOUT
+///
+/// NOTE: The ordering and format of the output is subject to change and
+/// should not be relied on to remain consistent over time.
+///
+/// # Examples
+///
+/// Enables the console perf event subscriber for the server,
+/// configured to print metrics every 10 seconds
+///
+/// ```rust,ignore
```

Comment:
This code block seems messed up: https://dnglbrstg7yg.cloudfront.net/b110450f4ad4aa35bf1f747baf46fd744aecceb8/doc/s2n_quic/provider/event/console_perf/struct.Subscriber.html#examples